config/examples/backend_redis.yml

```diff
@@ -1,29 +1,17 @@
-# Configuration for Redis. Separate per each environment.
-#
-# Accepted options are:
-#
-# servers       list of servers (hostname:port) to connect to. The first one is the master, the
-#               others are used for failover (no sharding yet).
-# db            database to use. A number from 0 to 10 (default is 0)
-# timeout
-# password
-# thread_safe
-base: &base
+base: &default
   url: "<%= ENV.fetch('BACKEND_REDIS_URL', 'redis://localhost:6379/0') %>"
+  timeout: <%= ENV.fetch('BACKEND_REDIS_TIMEOUT', 1) %>
   pool_size: <%= ENV.fetch('RAILS_MAX_THREADS', 5) %>
   pool_timeout: 5 # this is in seconds
   sentinels: "<%= ENV['BACKEND_REDIS_SENTINEL_HOSTS'] %>"
   role: <%= ENV['BACKEND_REDIS_SENTINEL_ROLE'] %>
 
-production:
-  <<: *base
-
-preview:
-  <<: *base
-
 development:
-  <<: *base
+  <<: *default
 
 test:
-  <<: *base
+  <<: *default
   db: <%= 2 + ENV['TEST_ENV_NUMBER'].to_i %>
+
+production:
+  <<: *default
```

Comment:
Here also I'm a little concerned to leave production use same config as development as confusion may happen between the two environments if one starts production locally for whatever reasons. But since it was like that before, I guess we can do in another PR if we want.

Reply:
Yes, I also thought that actually, `production` environment probably doesn't have much sense in these example files.
The only reason I can think of is that if somebody wants to try the app locally in "production mode" (because some things change in the behavior).

Certainly we can change this later.